pages/selfCheckQuestions.tsx

```diff
@@ -0,0 +1,284 @@
+import React from "react";
+import {
+    Spacer,
+    Text,
+    Modal,
+    ModalBody,
+    ModalContent,
+    ModalOverlay,
+    ModalHeader,
+    ModalFooter,
+    ModalCloseButton,
+    useDisclosure,
+    Flex,
+    SimpleGrid,
+    Button,
+} from "@chakra-ui/react";
+import { Header, SelfCheckQuestionCard, Footer } from "@components";
+
+//Default question list, can be shortened as needed
+const questionList = [
+    {
+        _id: "60e642d7e4a1ae34207a92a3",
+        type: "multiple_choice",
+        question: "autem sunt eiusdolores nesciunt impedit?",
+        options: ["1", "2", "3"],
+        alphanumericInput: true,
+        questionNumber: 1,
+    },
+    {
+        _id: "60e642d7e4a1ae34207a92a4",
+        type: "multi_select",
+        question: "autem sunt eiusdolores nesciunt impedit?",
+        options: ["1", "2", "3"],
+        alphanumericInput: true,
+        questionNumber: 2,
+    },
+    {
+        _id: "60e642d7e4a1ae34207a92a5",
+        type: "short_answer",
+        question: "autem sunt eiusdolores nesciunt impedit?",
+        options: ["1", "2", "3"],
+        alphanumericInput: true,
+        questionNumber: 3,
+    },
+    {
+        _id: "60e642d7e4a1ae34207a92a6",
+        type: "long_answer",
+        question: "autem sunt eiusdolores nesciunt impedit?",
+        options: ["1", "2", "3"],
+        alphanumericInput: true,
+        questionNumber: 4,
+    },
+    {
+        _id: "60e642d7e4a1ae34207a92a7",
+        type: "slider",
+        question: "autem sunt eiusdolores nesciunt impedit?",
+        options: ["1", "2", "3", "4", "5"],
+        alphanumericInput: true,
+        questionNumber: 5,
+    },
+];
+
+//Self Check Questions React functional component
+const Home: React.FC = () => {
+    const [list, setList] = React.useState(questionList);
+    const [count, setCount] = React.useState(questionList.length + 1);
+    const newSliderRange0 = [];
+    for (let i = 1; i <= 3; i++) {
+        newSliderRange0.push(i);
+    }
+    const [sliderRange, setSliderRange] = React.useState(newSliderRange0);
+    function changeQuestionType(id, target) {
+        const newList = list.slice(0);
+        newList[newList.findIndex((e) => e._id === id)].type = target;
+
+        setList(newList);
+    }
+    function changeAlphanumeric(id, target) {
```

Comment:
whats the point of this? I noticed in the UI that it didn't really change anything, am i missing something?

Reply:
It's mostly for the db, it tells us whether to store our values as an float or alphanumerically